### DISCUSSION TITLE
include output from pyviz visualization stack

### DISCUSSION THREAD

**Discussion by ea42gh** [quarto-dev/quarto-cli#1859] [Q&A]
Anybody here know a quick workaround to get this to work? (I did submit an issue in quarto-python)
---
title: "Quarto Basics"
format:
  html:
    code-fold: true
jupyter: python3
---
` ` `{python}
#| label: holoviews
#| fig-cap: "Holoviews plot"

import numpy as np
import holoviews as hv;
hv.extension('bokeh', logo=False);
 
r            = np.arange(0, 2, 0.01)
theta        = 4 * np.pi * r

hv.Curve((theta,r))
` ` `

**Comment by jjallaire** [comment]
I've created an issue to track this: https://github.com/quarto-dev/quarto-cli/issues/1867
